docs/src/index.md

```diff
@@ -64,8 +76,8 @@ a list of well-supported libraries that provide different data science tools,
 along with a few notes about what makes each library special, and how well
 integrated they are with DataFrames.jl.
 
-
-- **Statistics**
+```@raw html
```

Comment:
1. why this triplebackquote with `@raw html` is needed?
2. what does `??? tip` command do?

Reply:
`raw` and `???` are to have an expandable menu.